src/core/flex.ts

```diff
@@ -15,15 +15,19 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 import { assertTruthy } from '@lightningjs/renderer/utils';
-import type { ElementNode, SolidNode } from './node/elementNode.js';
+import {
+  type ElementNode,
+  type SolidNode,
+  NodeTypes,
+} from './node/elementNode.js';
 
 export default function (node: ElementNode): boolean {
   const children = [];
   let hasOrder = false;
   for (let i = 0; i < node.children.length; i++) {
     const c = node.children[i]!;
     // Filter empty text nodes which are place holders for <Show> and elements missing dimensions
-    if (c.name === 'TextNode') {
+    if (c.type === NodeTypes.Text) {
```

Comment:
A string union would have been slightly more efficient as you'd have kept `c.type === 'TextNode'